### ISSUE TITLE
The loss curve does not fall

### ISSUE THREAD

**Issue by luoshuifeiyang** [Ugness/PiCANet-Implementation#27]
I didn't change the network, but the loss didn't fall. I thought the lr was too big or small ,but even I set it bigger or smaller, the loss curve always not fall.  the model didn't  converge。 Could you please tell me the reason? Thanks a lot.

**Comment by Ugness** [comment]
1. If you train the network from the pretrained one, it would be possible that the loss didn't fall.
2. If you train the network from random initialization, please check your data in/out is correct (same name image file in each images / masks folder).

![image](https://user-images.githubusercontent.com/33034982/90327942-4f50a480-dfd3-11ea-8099-d9e9dbd8f83d.png)

This is my loss graph (from random initialization), and you can see that loss does not look converging about the first 10,000 steps. In my opinion, please wait for the model to converge, and please look carefully if the saliency prediction masks seem reasonable or not. If the masks do not seem reasonable, please attach the image of your saliency mask and loss graph to let me help you more.
